README.md

```diff
@@ -90,6 +90,41 @@ Please note that if you use a kubernetes cluster created on DigitalOcean, there
 will be a cloud controller manager running in the cluster already, so you local
 one will compete for API access with it.
 
+### Run Locally (optional parameters)
+
+In addition to the standard way of running `digitalocean-cloud-controller-manager`
+locally against a particular cluster, you can add some additional parameters that
+enable you to run the cluster with some extra features.
```

Comment:
If we call this entire section `Optional Features` I think we can get rid of this paragraph has a kind of filler feel to it.